app/views/cash_account_requested_too_many_transactions.scala.html

```diff
@@ -0,0 +1,44 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import viewmodels.RequestedTooManyTransactionsViewModel
+
+@this(
+        main_template: Layout,
+        h1: components.h1,
+        inset: components.inset,
+        link: components.link
+)
+
+@(
+        viewModel: RequestedTooManyTransactionsViewModel
+)(
+        implicit request: Request[_],
+        messages: Messages,
+        appConfig: AppConfig
+)
+
+@main_template(pageTitle = Some(messages("cf.cash-account.detail.title")),
```

Comment:
Please retrieve title from the viewmodel as it is already present in the viewmodel